pkg/ui/react-app/src/thanos/pages/blocks/Blocks.tsx

```diff
@@ -1,6 +1,6 @@
-import React, { FC, useMemo, useState } from 'react';
+import React, { FC, useEffect, useMemo, useRef, useState } from 'react';
```

Comment:
```suggestion
import React, { FC, useMemo, useRef, useState } from 'react';
```

Sorry if I missed it but I guess we are not using `useEffect` anywhere so let's remove the import.

Reply:
Oops I missed that, thanks!